sw/device/lib/dif/BUILD

```diff
@@ -34,7 +34,6 @@ cc_test(
         "autogen/dif_clkmgr_autogen_unittest.cc",
         "dif_clkmgr.c",
         "dif_clkmgr.h",
-        "dif_clkmgr_unittest.cc",
```

Comment:
(just to document this, as I'm sure you are aware) you can add this back in when #8822 merges.